.github/workflows/rust.yml

```diff
@@ -0,0 +1,22 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Avoid using ubuntu-latest since this will change when GitHub upgrade their runners to a latest Ubuntu version. Instread lock it to 22.04.